js-intermediate.js

```diff
@@ -44,41 +64,63 @@ var originalArray1 = [1, 2, 3, 4, 5, 6, 7]
 // Expected output: [7, 6, 5, 4, 3, 2, 1],
 var originalArray2 = ["a", "t", "l", "e", "d"]
 // Expected output: ["d", "e", "l", "t", "a"]
-
-
+let reverseArr = (arrType) => {
+  return arrType.reverse()
+}
+console.log(reverseArr(originalArray1))
+console.log(reverseArr(originalArray2))
 
 // 5. Write the code that would make this test pass.
 
-describe("multTwo", () => {
-    test ("returns an array with all the numbers multiplied by two", () => {
-        expect(multTwo([3, 4, 5])).toEqual([6, 8, 10])
-        expect(multTwo([23, -9, 0])).toEqual([46, -18, 0])
-        expect(multTwo([4.5, -4.5, 12])).toEqual([9, -9, 24])
-    })
-})
+// describe("multTwo", () => {
+//     test ("returns an array with all the numbers multiplied by two", () => {
+//         expect(multTwo([3, 4, 5])).toEqual([6, 8, 10])
+//         expect(multTwo([23, -9, 0])).toEqual([46, -18, 0])
+//         expect(multTwo([4.5, -4.5, 12])).toEqual([9, -9, 24])
+//     })
+// })
+// let multTwo = (arr) => {
+//   let splArr = ...arr
+//   return splArr * 2
+// }
+// console.log(multTwo([3, 4, 5]))
 
 
 
 // 6. Write a function called letterCounter that logs the number of times the letter "l" appears in ourString.
 
 var ourString = "Hello Learn Students!"
 // Expected output: 3
-
-
+let findL = (str) => {
+  let newLArr = []
+  let newStr = str.toLowerCase().split('')
+    for (let i = 0; i < str.length; i++)
+    if(newStr[i] === 'l'){
+       newLArr.push(newStr[i])
+    }
+    return newLArr.length;
+}
+console.log(findL(ourString))
 
 // 7. Create a function that takes in a string of a single word and returns the middle letter of the word. If the word is an even number of letters, return the two middle letters.
 
 var middleLetters1 = "hello"
 // Expected output: “l”
-var middleLetters2 = "lamas"
+var middleLetters2 = "llamas"
 // Expected output: “am”
-
+let findMiddle = (letters) => {
+  if(letters.length % 2 !== 0){
+    
+  } else {
+    
+  }
+}
 
 
 
 // 8. Write a program to get the area of a sphere using object classes. Create three spheres with different radi as test cases. Area of a sphere =  4πr^2 (four pi r squared)
 
-
+// i am havin trouble starting this program
 
```

Comment:
class Sphere{
  constructor(radius){
  this.radius = radius
  }
  getArea(){
    let pi = Math.PI
    return 4*pi*this.radius**2
  }
}
var mySphere = new Sphere(6)

console.log(mySphere.getArea())